packages/driver/src/multi-domain/serializer.ts

```diff
@@ -0,0 +1,39 @@
+import { isFunction, isSymbol } from 'lodash'
+
+/**
+ * serialize. Used for serializing subject to be returned from subdomains to the primary domain.
+ * @param value any object to be serialized.
+ * @returns the serialized object as a string or undefined, if the object cannot be serialized.
+ */
+const serialize = (value: any): string => {
```

Comment:
@AtofStryker, I know that you put together some level of serialization for the errors but this seemed different enough? Perhaps I should rename this `subjectSerializer`?

Reply:
I just saw this comment after review. Yeah I think that is a good idea to distinguish between the two! I have a feeling we are going to have to address serialization pretty soon since it sounds like we are going to have quite a few and might need to see where we can consolidate on.